NiceIO.cs

```diff
@@ -233,6 +253,10 @@ public override string ToString()
 
 		public string ToString(SlashMode slashMode)
 		{
+			// Check if it's linux root /
+			if (_elements.Length == 0 && !_isRelative && string.IsNullOrEmpty(_driveLetter))
```

Comment:
let's use .IsRoot && driveletter check here.   can we use == null for the drivelettercheck instead of nullorempty?